backend/src/v5/middleware/dataConverter/inputs/teamspaces/projects/models/commons/groups.js

```diff
@@ -47,7 +60,10 @@ Groups.validateGroupsImportData = async (req, res, next) => {
 	try {
 		const { groups } = req.body;
 		if (!groups?.length) throw createResponseCode(templates.invalidArguments, 'Groups array cannot be empty');
-		await Promise.all(groups.map(validateSchema));
+		await Promise.all(groups.map((g) => {
```

Comment:
when you pass strict = true the transformations are not executed. But if I pass strict = false it accepts different data types that required ... for example it would accept values : [1,2,3] in STARTS_WITH and convert them to strings thats why I thought this is the only solution